charts/argo-rollouts/values.yaml

```diff
@@ -12,8 +10,9 @@ controller:
   tolerations: []
   affinity: {}
   image:
+    registry: quay.io
     repository: argoproj/argo-rollouts
-    tag: v0.10.2
+    tag: ""
```

Comment:
Shall we provide a default value, say 1.0.0?

Reply:
I moved it to the `Charts.yaml` so we dont have to maintain it in Charts.yaml, README and values.yaml.